app/preload/ledger/ledgerObserverImpl.ts

```diff
@@ -7,12 +7,23 @@ import type {
     Subscription,
 } from '@ledgerhq/hw-transport';
 import EventEmitter from 'events';
+import { usb } from 'usb';
+import { identifyUSBProductId, ledgerUSBVendorId } from '@ledgerhq/devices';
 import ConcordiumLedgerClientMain from '../../features/ledger/ConcordiumLedgerClientMain';
 import { isConcordiumApp, isOutdated } from '../../components/ledger/util';
 import { LedgerSubscriptionAction } from '../../components/ledger/useLedger';
 import ledgerIpcCommands from '~/constants/ledgerIpcCommands.json';
 import { LedgerObserver } from './ledgerObserver';
 
+/**
+ * A convenience method for opening a transport by providing
+ * an empty descriptor (as the descriptor is unused downstream).
+ * @returns a promise with a {@link TransportNodeHid}
+ */
+async function openTransport(): Promise<TransportNodeHid> {
+    return TransportNodeHid.open('');
```

Comment:
This has started throwing errors (assumeditably because of dependency update), could we handle those?

Reply:
I have refactored a bit to better catch the errors that may be thrown.